src/iscan.cpp

```diff
@@ -100,23 +100,40 @@ usage() {
     char *msgs[] = {
         "Usage: iscan [-rhd] srcPhysicalFile|srcPhysicalDirectory|srcDataObj|srcCollection",
         " ",
-        "If the input is a local data file or a local directory, iscan",
-        "checks if the content is registered in iRODS.",
+        "If the input is a local physical data file or a local directory,",
+        "iscan checks if the named content is registered in iRODS.",
+        "A full physical path must be used for local files and directories.",
         " ",
-        "Full path must be used for local files and directories.",
-        " ",
-        "If the input is an iRODS file or an iRODS collection, iscan",
-        "checks if the physical files corresponding to the iRODS object",
-        "exists on the data servers. Scanning data objects and",
-        "collections may only be performed by a rodsadmin.",
+        "If the input is a logical path representing an iRODS data object",
+        "or an iRODS collection, iscan checks if the associated registered",
+        "physical file entries exist on the data servers.",
+        "Scanning data objects and collections may only be performed by a rodsadmin.",
         " ",
         "If the operation is successful, nothing will be output",
-        "   and 0 will be returned.",
+        "and 0 will be returned.",
         " ",
         "Options are:",
-        " -r  recursive - scan local subdirectories or subcollections",
+        " -r  recursive - scan local subdirectories or logical subcollections",
         " -h  this help",
-        " -d  scan data objects in iRODS (default is scan local objects)",
+        " -d  scan data objects in iRODS (default is scan local files/directories)",
+        " ",
+        "Examples:",
+        "$ iscan /var/lib/irods/notincatalog",
+        "/var/lib/irods/notincatalog is not registered in iRODS",
```

Comment:
Inconsistent naming between `notincatalog` and `not_a_dataobject` (on L131).

Reply:
ah, updated.